lib/plugins/rest.js

```diff
@@ -4,10 +4,21 @@ var InstanceAdapter = require("../services/instanceAdapter.js");
 var Boom            = require("boom");
 var _               = require("lodash");
 var Joi             = require("joi");
+var AwsAdapter      = require("../services/awsAdapter");
+var AWS             = require("aws-sdk");
+var Bluebird        = require("bluebird");
+
+var ec2 = new AWS.EC2({
```

Comment:
Any reason to have this intialization logic run at the start, rather than in the `register` setup?